support-frontend/assets/pages/contributions-landing/components/ContributionTypeTabs.jsx

```diff
@@ -67,6 +69,36 @@ const mapDispatchToProps = (dispatch: Function) => ({
   },
 });
 
+const tabGroupStyles = (checkedContributionType: ContributionType) => css`
+  > div {
+    display: flex;
+
+    label:nth-of-type(1) {
+      border-top-right-radius: 0;
+      border-bottom-right-radius: 0;
+      /* This is to override a margin that is added by source */
+      margin-right: 0;
+
+      z-index: ${checkedContributionType === 'ONE_OFF' ? 100 : 1}
```

Comment:
This is slightly awkward, but we need the currently selected amount to be on top of the others so that the margins don't get cut off when highlighted